files/ui/advancedpage.ui

```diff
@@ -454,6 +454,16 @@
                 </property>
                </widget>
               </item>
+              <item row="3" column="1">
+               <widget class="QCheckBox" name="antialiasAlphaTestCheckBox">
+                <property name="toolTip">
+                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows MSAA to work with alpha-tested meshes, producing better-looking edges without pixelation. Can negatively impact performance if MSAA is off.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
I'd get rid of the *if MSAA is off* - it'll also impact performance at least a tiny bit when it's on and I think you decided we'd probably actually made it not impact performance when it's off.

Reply:
Does it look good now?

Reply:
Yep